README.md

```diff
@@ -40,9 +40,9 @@ $twitch_client_id = 'TWITCH_CLIENT_ID';
 $twitch_client_secret = 'TWITCH_CLIENT_SECRET';
 $twitch_scopes = '';
 
-$helixGuzzleClient = new \NewTwitchApi\HelixGuzzleClient($twitch_client_id);
-$newTwitchApi = new \NewTwitchApi\NewTwitchApi($helixGuzzleClient, $twitch_client_id, $twitch_client_secret);
-$oauth = $newTwitchApi->getOauthApi();
+$helixGuzzleClient = new \TwitchApi\HelixGuzzleClient($twitch_client_id);
+$TwitchApi = new \TwitchApi\TwitchApi($helixGuzzleClient, $twitch_client_id, $twitch_client_secret);
```

Comment:
Please keep the variable names starting with lowercase. I'm assuming this is simply the result of a quick and easy find and replace. :-)

Reply:
Good catch, thank you!